frontend/public/components/conditions.tsx

```diff
@@ -1,11 +1,11 @@
 import * as React from 'react';
-import * as _ from 'lodash-es';
 
 import { Timestamp } from './utils';
 import { CamelCaseWrap } from './utils/camel-case-wrap';
+import { K8sResourceCondition } from '../module/k8s';
 
-export const Conditions: React.SFC<ConditionsProps> = ({ conditions }) => {
-  const rows = _.map(conditions, (condition, i) => (
+export const Conditions: React.SFC<ConditionsProps> = ({ conditions = [] }) => {
+  const rows = conditions.map((condition: K8sResourceCondition, i: number) => (
```

Comment:
This is a little safer than `conditions = []` since it would handle `null` values as well.

```suggestion
  const rows = (conditions || []).map((condition: K8sResourceCondition, i: number) => (
```

Reply:
I'd suggest changing line 31 to `conditions?.length ? ...` as well